open-sphere-base/mantle/src/main/java/io/opensphere/mantle/data/MapVisualizationType.java

```diff
@@ -73,10 +73,6 @@
     /** The type used for visualizing tiles composed of interpolated data. */
     INTERPOLATED_IMAGE_TILES(MapVisualizationStyleCategory.IMAGE);
 
-    /** Predicate for testing if this is a map data element type. */
-    public static final Predicate<MapVisualizationType> IS_MAP_DATA_ELEMENT_TYPE_PREDICATE = type -> type != null

```

Comment:
Why remove this, but leave the other?

Reply:
Its only use was in the other class I changed.  I wasn't touching code that used the other predicate.